conjure-java-jaxrs-client/build.gradle

```diff
@@ -8,26 +8,26 @@ dependencies {
     api "com.google.code.findbugs:jsr305"
     api "javax.ws.rs:javax.ws.rs-api"
     // TODO(dsanduleac): Should be implementation, but can't because we expose feign.TextDelegateEncoder
-    api "com.netflix.feign:feign-core"
+    api "io.github.openfeign:feign-core"
```

Comment:
@pkoenig10 I think the TODO above might be out of date... afaik the reason this has to be api is because people may want to catch a `RetryableException`

Reply:
Let's update it in a follow up. Fixing the TODO is unrelated to this upgrade.